.github/workflows/pythonpackage.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ["3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.9", "3.10", "3.11"]
```

Comment:
We are required to drop Python 3.12 as Numba 0.58 doesn't support it yet. Python 3.12 _should_ be supported in the next release. This should be released in early 2024.